### PR TITLE
support aws sessionToken

### PR DESCRIPTION
**Pull request by xquanluu**
https://github.com/jambonz/jambonz-feature-server/issues/726

### CODE REVIEW

**Comment by davehorton** [comment]
is this tested?

**Comment by xquanluu** [comment]
yes, it's tested by me on customer jambonz mini system

File: mod_aws_lex/aws_lex_glue.cpp [hunk 1]
```diff
@@ -190,8 +189,11 @@ class GStreamer {
 		for (int i = 4; i < 20; i++) keySnippet[i] = 'x';
 		keySnippet[19] = '\0';
 
-		switch_log_printf(SWITCH_CHANNEL_LOG, SWITCH_LOG_DEBUG, "GStreamer %p ACCESS_KEY_ID %s\n", this, keySnippet);		
-		if (*awsAccessKeyId && *awsSecretAccessKey) {
+		switch_log_printf(SWITCH_CHANNEL_LOG, SWITCH_LOG_DEBUG, "GStreamer %p ACCESS_KEY_ID %s\n", this, keySnippet);
+		if (*awsAccessKeyId && *awsSecretAccessKey && *awsSessionToken) {
+			switch_log_printf(SWITCH_CHANNEL_LOG, SWITCH_LOG_DEBUG, "using AWS creds %s %s %s\n", awsAccessKeyId, awsSecretAccessKey, awsSessionToken);	
+			m_client = Aws::MakeUnique<LexRuntimeV2Client>(ALLOC_TAG, AWSCredentials(awsAccessKeyId, awsSecretAccessKey, awsSessionToken), config);
+		}else if (*awsAccessKeyId && *awsSecretAccessKey) {
```
Comment (davehorton) [inline mod_aws_lex/aws_lex_glue.cpp:196]: can you add a space after the brace

**Review by davehorton — CHANGES_REQUESTED** [review]

File: mod_aws_lex/aws_lex_glue.cpp (hunk 1 above)
Comment (xquanluu) [reply mod_aws_lex/aws_lex_glue.cpp]: fixed

**Review by davehorton — APPROVED** [review]

**Pull request merged**